experimental/GroebnerWalk/src/GroebnerWalk.jl

```diff
@@ -0,0 +1,19 @@
+module GroebnerWalk
+using Oscar
+import Oscar: weight_ordering, ZZRingElem, MonomialOrdering, ZZMatrix, IdealGens
+import Oscar.Orderings: MatrixOrdering, _support_indices
+
+include("markedGB.jl")
+include("common.jl")
+
+include("standard_walk.jl")
+include("generic_walk.jl")
+include("perturbed_walk.jl")
+
+export groebner_walk
+
+function __init__()
+    add_verbosity_scope(:groebner_walk)
+end
+
+end
```

Comment:
```suggestion
end

export groebner_walk
```
If you want to have functions available for Oscar users, you need to export them twice (once inside the module, and once at the bottom of the file).
Once you do this, all `using Oscar.GroebnerWalk` should be unnecessary.